website/www/site/content/en/contribute/get-started-contributing.md

```diff
@@ -0,0 +1,383 @@
+1. Create an account on [Beam issue tracker (JIRA)](https://issues.apache.org/jira/projects/BEAM/issues)
+   (anyone can do this).
+
+### Share your intent
+
+1. Find or create an issue in the [Beam issue tracker (JIRA)](https://issues.apache.org/jira/projects/BEAM/issues).
+   Tracking your work in an issue will avoid duplicated or conflicting work, and provide
+   a place for notes. Later, your pull request will be linked to the issue as well.
+1. If you want to get involved but don't have a project in mind, check our list of open starter tasks,
+   [https://s.apache.org/beam-starter-tasks](https://s.apache.org/beam-starter-tasks).
+1. Assign the issue to yourself. To get the permission to do so, email
+   the [dev@ mailing list](/community/contact-us)
+   to introduce yourself and to be added as a contributor in the Beam issue tracker including your
+   ASF Jira Username. For example [this welcome email](https://lists.apache.org/thread.html/e6018c2aaf7dc7895091434295e5b0fafe192b975e3e3761fcf0cda7@%3Cdev.beam.apache.org%3E).
+1. If your change is large or it is your first change, it is a good idea to
+   [discuss it on the dev@ mailing list](/community/contact-us/).
+1. For large changes create a design doc
+   ([template](https://s.apache.org/beam-design-doc-template),
+   [examples](https://s.apache.org/beam-design-docs)) and email it to the [dev@ mailing list](/community/contact-us).
+
+### Development Setup {#development-setup}
+
+1. Check [Git workflow tips](https://cwiki.apache.org/confluence/display/BEAM/Git+Tips) if you need help with git forking, cloning, branching, committing, pull requests, and squashing commits.
+1. Clone the git repository. You can download it anywhere you like.
+
+       $ mkdir -p ~/go/src/github.com/apache
+       $ cd ~/go/src/github.com/apache
+       $ git clone https://github.com/apache/beam
+       $ cd beam
+
+       - For Go development:
+       We recommend putting it in your [`$GOPATH`](https://golang.org/doc/gopath_code#GOPATH) (`$HOME/go` by default on Unix systems).
+          1. Clone the repo, and update your branch as normal
+                  $ git clone https://github.com/apache/beam.git
+                  $ cd beam
+                  $ git remote add <GitHub_user> git@github.com:<GitHub_user>/beam.git
+                  $ git fetch --all
+          1. Get or Update all the Go SDK dependencies
+                  $ go get -u ./...
+
+1. Check the environment was set up correctly.
+  - **Option 1**: validate the Go, Java, and Python environments:
+
+    **Important**: Make sure you have activated Python development.
+```
+./gradlew :checkSetup
+```
+    - **Option 2**: Run independent checks:
+        - For **Go development**:
+          1. Execute:
+```
+export GOLANG_PROTOBUF_REGISTRATION_CONFLICT=ignore
+./gradlew :sdks:go:examples:wordCount
+```
+         - For **Python development**:
+          1. Execute:
+```
+./gradlew :sdks:python:wordCount
+```
+        - For **Java development**:
```

Comment:
It is changed in PR#17098 (Beam 13992) https://github.com/apache/beam/pull/17089. 
13992 should be merged before this PR, because it contains changes, that will be used in current PR, as it is stated in description